source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/source/road/ADMSRoadSideBarrierType.java

```diff
@@ -21,9 +21,29 @@
  */
 public enum ADMSRoadSideBarrierType {
   NONE,
-  NOISE_BARRIER,
-  STREET_CANYON_TERRACED_HOUSES,
-  STREET_CANYON_DETACHED_HOUSES,
-  TREE_BARRIER,
+  NOISE_BARRIER(5D),
+  BRICK_WALL(5D),
+  STREET_CANYON_TERRACED_HOUSES(20D),
+  STREET_CANYON_SEMIDETACHED_HOUSES(40D),
+  STREET_CANYON_DETACHED_HOUSES(50D),
+  TREE_BARRIER_OPEN(80D),
+  TREE_BARRIER_DENSE(90D),
   OTHER;
+
+  private Double defaultPorosity;
```

Comment:
Does it make sense to use a object instead of primitive? You could use `Double.NaN` for `Other`?

Reply:
We could use primitive, the only real effect would be having to write

```java
if (barrierType.getDefaultPorosity() != Double.NaN) {
  setBarrierLeftPorosity(String.valueOf(v));
}
```

instead of 

```java
Optional.ofNullable(barrierType.getDefaultPorosity())
  .ifPresent(v -> setBarrierLeftPorosity(String.valueOf(v)));
```

I think the latter is cleaner personally

Reply:
Ah yes. Still feels strange to not use primitive, and have to rely on null checks. Another option could be to add an extra method `hasPorosity()`.